### PR TITLE
add Gitter chat badge

### PR DESCRIPTION
**Pull request by ethanhs**

### CODE REVIEW

**Comment by gvanrossum** [comment]
I'd like Matthias and someone from PyCharm to agree. Also maybe add something to CONTRIBUTING.md?

**Comment by ethanhs** [comment]
There is already [a mention](https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#discussion) in CONTRIBUTING.md.

**Comment by matthiaskramm** [comment]
Typeshed is a generic project, and mypy is just one of multiple projects that use it.

We should start a chat channel about general Python static typing. And then also link to it from http://github.com/python/typing etc.
@ethanhs, is this something you'd want to look into?

**Comment by ethanhs** [comment]
I made this PR at Guido's request as I had mentioned I wasn't sure if mypy's was the best place for discussion of some question. Guido had said he didn't want to fracture the chats too much, saying
>I'm a little worried about prematurely spitting this forum up between mypy, typeshed and typed_ast. It's not like there's so much traffic, often it's unclear to users which one is the source of the problem, and it's mostly the same people helping anyways.

But I do like the idea of having a general static typing channel (which mypy seems to be serving at the moment). But I don't feel strongly on the matter and it doesn't matter too much to me.

**Comment by matthiaskramm** [comment]
I'm fine with having just one channel for all things typing. I'd just recommend we name it appropriately, so users are not confused.

**Comment by gvanrossum** [comment]
OK, so we could have a single python/typing channel, or we could have two channels, python/mypy and python/typing. I'm not sure which is better -- two channels would dilute developer attention, but one would likely be full of mypy-specific chatter, and I expect the mypy channel would still have a lot of typeshed questions (the mypy tracker gets lots of typeshed bug reports). If Matthias is okay with having the mypy traffic I'd prefer a single python/typing channel; if not, let's have two channels (and of course you can also create a channel for pytype).

I'm curious whether PyCharm currently has a chat channel at all -- @vlasovskikh?

**Comment by matthiaskramm** [comment]
Yes, I'm happy with (and would expect) mypy-chatter in the typing channel. mypy's the project with, currently, the biggest traction.

**Comment by gvanrossum** [comment]
OK, let's all move to python/typing. I've created the channel and invited a few people.

@ethanhs can you update this PR? Also update CONTRIBUTING.md. Then we also need to update the mypy project (if @JukkaL agrees).

**Comment by ethanhs** [comment]
Okay this should be good to go.

File: README.md [hunk 1]
```diff
@@ -101,7 +103,7 @@ https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#stub-versioning).
 ## Contributing
 
 Please read [CONTRIBUTING.md](CONTRIBUTING.md) before submitting pull
-requests.
+requests. If you have questions related to contributing, drop by the [mypy Gitter](https://gitter.im/python/mypy).
```
Comment (gvanrossum) [inline README.md]: Actually the plan is to completely kill the python.mypy gitter room, so this should also point to python/typing.

**Comment by gvanrossum** [comment]
Thanks!

**Pull request merged**